docs/workloads/azure/backend/java/scaffolding_java.md

```diff
@@ -22,30 +22,43 @@ import ResponsivePlayer from "../../../../../src/pages/Components/ResponsivePlay
 
 <ResponsivePlayer url="https://vimeo.com/640390929" />
 
-
-**Before starting, please make sure that your environment meets all the [requirements](requirements_java.md).**
+:::caution
+Before starting, please make sure that your environment meets all the [requirements](requirements_java.md).
+:::
 
 You can customize the deployment of the project running a handful of command lines. Currently, scaffolding is handled by Maven and you can use it
-by cloning one of the repositories and then installing the project as a Maven archetype locally; from there we can create a new project by using
+by cloning one of the workload repositories and then installing the project as a Maven archetype locally; from there, we can create a new project by using
 that archetype as a template. To do so we do:
 
-1. Clone any of the STACKS Java project (simple-api, stacks-java-cqrs, stacks-java-cqrs-events)
+1. Clone any of the Stacks Java Workloads (`simple-api`, `stacks-java-cqrs` or `stacks-java-cqrs-events`)
 2. Navigate to the `<directory you cloned the project into>/java` in the terminal
 3. Then issue the following Maven commands, using the included wrapper:
-   1. ``./mvnw archetype:create-from-project`` - To create the archetype
-   2. `` cd target/generated-sources/archetype`` - Navigate to the folder it was crated in
+   1. ``./mvnw archetype:create-from-project -DpropertyFile=archetype.properties`` - To create the archetype
```

Comment:
@guillermogutierrez / @johnsall : is this still required?

Reply:
Yes it is, the properties file includes file extensions and exclusions